nautilus_trader/common/actor.pyx

```diff
@@ -2570,13 +2572,10 @@ cdef class Actor(Component):
             The instrument ID for the request.
         start : datetime, optional
             The start datetime (UTC) of request time range.
-            If ``None`` then will be replaced with the current UTC time.
-            Should be left-inclusive (start <= value), but inclusiveness is not currently guaranteed.
-            The `start == end` case is allowed to request only the initial state.
+            The inclusiveness depends on individual data client implementation.
```

Comment:
I think these are more accurate param descriptions right now. We have not fully committed to consistent inclusiveness across all integrations, and are also not achieving it just yet even if that might be a future goal.